cmd/kube-apiserver/app/server.go

```diff
@@ -210,9 +212,16 @@ func Run(s *options.APIServer) error {
 				installSSH = instances.AddSSHKeyToAllInstances
 			}
 		}
-
+		if ports.KubeletReadOnlyPort <= 0 {
```

Comment:
Isn't this just checking that a constant (compiled in) was defined? I'm not sure what value this gives other than causing the kube-apiserver to not compile / crash if we ever remove or change that value. Was that the intent of this statement?